### ISSUE TITLE
pandas: Incorrect return type for `pandas.Timestamp.normalize()`

### ISSUE THREAD

**Issue by DannyNemer** [microsoft/pylance-release#779]
## Environment
-   Language Server: v2020.12.2
-   OS: macOS v10.15.6
-   Python: v3.7.9
-   Pandas: v1.1.5 (latest)

## Expected behavior

Recognize that [`pandas.Timestamp.normalize()`](http://pandas.pydata.org/docs/reference/api/pandas.Timestamp.normalize.html#pandas.Timestamp.normalize) returns an instance of `pandas.Timestamp`:

<img width=400 src=https://user-images.githubusercontent.com/170023/103165861-c5f77d00-47d1-11eb-93ee-b9ef3a48a68a.png>

## Actual behavior

Pylance incorrectly says `pandas.Timestamp.normalize()` returns `None`:

<img width=500 src=https://user-images.githubusercontent.com/170023/103165868-dd366a80-47d1-11eb-9b3f-0c900a2e584f.png>

## Code Snippet

Should not return errors:
```python
import pandas as pd

pd.Timestamp.today().normalize().tz_localize(None)
```

---

Please let me know how I can improve this bug report because many more will follow (all for typeshed issues). Is this the best repo for such bug reports? (I am following the precedent of Pandas type issues in the release notes.)

**Comment by judej** [comment]
Thanks for the report @DannyNemer, looks like the type stub needs an update.

**Comment by gramster** [comment]
It doesn't seem like normalize is the issue here, as it says:

```
    def normalize(self, *args, **kwargs) -> Timestamp: ...
```

**Comment by gramster** [comment]
Interestingly this doesn't see to be reproing anymore; it was a week ago.

**Comment by gramster** [comment]
I also see "argument missing for parameter tz" on the line

```
pd.Timestamp.today()
```

which makes me wonder if `@classmethod` is being handled in the stubs.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.5.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202150-5-may-2021
